scripts/configurator/parser/cmake_var.py

```diff
@@ -1,9 +1,108 @@
-from typing import Dict, Any, Type, Mapping
+from typing import Dict, Any, Optional, Type, Mapping, List
+import re
+from sys import stderr
+
+
+class metadata:
+   keyword_expr = re.compile(r"([A-Z_]+)")
+
+   def __init__(self, comment: List[str]) -> None:
+      self.depends_on: Optional[List[str]] = None
```

Comment:
In a follow-up change, we'll have to check somewhere that the list in `depends_on` consists in actual bool variables. If the variables do not exist or they are not booleans, we have to show an error and fail.

Reply:
Ok. I suppose we will put this in another commit, right?

Reply:
Sure.

Reply:
Perfect, marking this as closed